Bio/Cluster/__init__.py

```diff
@@ -278,9 +278,9 @@ def treecluster(
        - method == 'a': Average pairwise linkage
      - distancematrix:  The distance matrix between the items. There are
        three ways in which you can pass a distance matrix:
-       1. a 2D numpy array (in which only the left-lower part of the array
+       1. a 2D Numpy array (in which only the left-lower part of the array
        will be accessed);
-       2. a 1D numpy array containing the distances consecutively;
+       2. a 1D Numpy array containing the distances consecutively;
```

Comment:
If changing numpy in the text, use NumPy not Numpy.

(4 cases in this file, lines 191, 193, 281 and 283)